client/me/account/main.jsx

```diff
@@ -87,6 +90,9 @@ const INTERFACE_FIELDS = [
 	'calypso_preferences',
 ];
 
+const BetaBadge = styled( Badge )`
```

Comment:
didn't feel right creating this styled component just for margins but inline styling of the `<Badge/>` component doesn't seem to work. maybe the component itself needed to modified to accept style props.

Reply:
The inline style wouldn't work because `<Badge>` doesn't forward any `style` prop through to its children. However it _does_ forward the `className` prop, which is why Emotion works.

The accounts page doesn't use Emotion, so I'd be reluctant to add it just for this one margin. If you take a look at the existing code in the `<Account>` component you'll see that they're applying CSS classes—which have been defined in `client/me/account/style.scss`—directly to elements.

So the thing to do here would be to define a new `account__beta-badge` class in `style.scss` and then apply that class directly to the `<Badge>` component in the render method. No need to define a new component.

Reply:
yeah defining the class make sense. Including emotion just didn't feels right.